### PR TITLE
Improvement of  webui doc

### PR DESCRIPTION
**Pull request by lvybriage**
#1391

### CODE REVIEW

File: docs/en_US/Tutorial/WebUI.md [hunk 1]
```diff
@@ -42,8 +42,8 @@ Click the tab "Intermediate Result" to see the lines graph.
 
 ![](../../img/webui-img/trials_intermeidate.png)
 
-The graph has a filter function. You can open the filter button. And then enter your focus point
-in the scape input. Simultaneously, intermediate result inputs can limit the intermediate's range.
+We set a filter function for the intermediate result because that your trials may have many intermediate results.
+You can use it if you want to see the trend of some trial. Import your focus intermediate result's step in the step input and then import the metric's min and max value you focus. Final, you can filter some trials to see the trend.
```
Comment (suiguoxin) [inline docs/en_US/Tutorial/WebUI.md]: Do you mean "input" other than "import" ?

Comment (lvybriage) [reply docs/en_US/Tutorial/WebUI.md]: Maybe input is better.

File: docs/en_US/Tutorial/WebUI.md [hunk 2]
```diff
@@ -42,8 +42,8 @@ Click the tab "Intermediate Result" to see the lines graph.
 
 ![](../../img/webui-img/trials_intermeidate.png)
 
-The graph has a filter function. You can open the filter button. And then enter your focus point
-in the scape input. Simultaneously, intermediate result inputs can limit the intermediate's range.
+We set a filter function for the intermediate result because that your trials may have many intermediate results.
```
Comment (xuehui1991) [inline docs/en_US/Tutorial/WebUI.md]: Can you explain how to set the three parameters?
Such as "Scape", "min" - "max" ??

Comment (lvybriage) [reply docs/en_US/Tutorial/WebUI.md]: Ok.

File: docs/en_US/Tutorial/WebUI.md [hunk 3]
```diff
@@ -44,11 +44,11 @@ Click the tab "Intermediate Result" to see the lines graph.
 
 We set a filter function for the intermediate result graph because that the trials may have many intermediate results in the training progress. You need to provide three data if you want to use the filter button to see the trend of some trial.
 
-What data should be written in the first input? Maybe you find a step those trials became better or worse. In other word, it's an important and concerned step. Just input it into the first input.
+What data should be written in the first input? Maybe you find an epoch those trials became better or worse. In other word, it's an important and concerned epoch. Just input it into the first input.
 
-After selecting the step, you should input your focus metric's range on this step. Yes, it's the min and max value. Like this picture, I choose the step is 13 and the metric's range is 20-60.
+After selecting the epoch, you should input your focus metric's range on this epoch. Yes, it's the min and max value. Like this picture, I choose the epoch is 13 and the metric's range is 20-60.
```
Comment (chicm-ms) [inline docs/en_US/Tutorial/WebUI.md]: 'epoch' could be more misleading than 'step'...

Comment (lvybriage) [reply docs/en_US/Tutorial/WebUI.md]: I set the placeholder "epoch" in the input...

**Pull request merged**